pkgs/development/libraries/gaia/default.nix

```diff
@@ -0,0 +1,85 @@
+{ stdenv
+, lib
+, fetchFromGitHub
+, libyaml
+, swig
+, pkgconfig
+, wafHook
+, makeWrapper
+, qt4
+, pythonPackages
+, pythonSupport ? false
+# Default to false since it breaks the build, see https://github.com/MTG/gaia/issues/11
+, stlfacadeSupport ? false
+, assertsSupport ? true
+, cyclopsSupport ? true
+}:
+
+assert pythonSupport -> pythonPackages != null;
+
+stdenv.mkDerivation rec {
+  pname = "gaia";
+  version = "2.4.5";
+
+  src = fetchFromGitHub {
+    owner = "MTG";
+    repo = "gaia";
+    rev = "v${version}";
+    sha256 = "12jxb354s2dblr2ghnl3w05m23jgzvrrgywfj8jaa32j3gw48fv2";
+  };
+
+  # Fix installation error when waf tries to put files in /etc/
+  prePatch = ''
+  '' + lib.optionalString cyclopsSupport ''
+    substituteInPlace src/wscript \
+      --replace "/etc/cyclops" "$out/etc/cyclops" \
+      --replace "/etc/init.d" "$out/etc/init.d"
+  '';
```

Comment:
BTW @aszlig, what do you think about the fact the package installs `init.d` files? I've had a look at these files today and I noticed they refer to paths which rely upon the fact the system will comply to the FHS hierarchy. Patching them might require some work. I wouldn't mind dive into it but NixOS doesn't even use these init files so what would be the point of that? Should we provide systemd files that will provide the same functionality? TBH I don't even know what these daemons are for and I've never used them not I intend to. Maybe we can keep it as is so future users of this package will be able to look at them for reference?